css/tech.scss

```diff
@@ -50,21 +54,26 @@
   }
 
   .section-2 {
+
```

Comment:
Sorry I wasn't clear before, but if there are no styles in a class you don't need to add a new line before starting another class.  So here you wouldn't need the new line, but if you have something like:

```
.hero {
  // styles
  
  .picture {
   // styles
  }
}
```

Then you would need a new line after the last style of `.hero` and `.picture`. Does that make sense?